projects/js-packages/charts/src/components/bar-chart/stories/index.docs.mdx

```diff
@@ -294,7 +293,7 @@ Display series information with automatic color and pattern matching:
 
 Control legend placement with alignment options:
 
-<Canvas of={ LegendStories.AlignmentPositioning } />
+<Canvas of={ BarChartStories.CustomLegendPositioning } />
 
 <Source
 	language="jsx"
```

Comment:
The canvas and source don't match for this, ie. the legend is not right aligned, which I now is a bug right now, so maybe the legend should not be in vertical orientation

![Screenshot 2025-08-05 at 12 53 55 PM](https://github.com/user-attachments/assets/840a336c-3c02-4b09-8804-7c3c3a4c439d)